sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -348,7 +360,23 @@ struct graphs_support;
 } // namespace device
 } // namespace info
 
-class node {};
+
+class node {
+  public:
+  class dynamic_parameter{
+    dynamic_parameter() = delete;
+    const std::string& getLabel() const;
+  };
+
```

Comment:
Only other helper function I think it's worth mentioning for the mixed explicit & record+replay case is, if we want to be able to get the node associated with the event returned from a record and replay event. Something like `static node getNodeFromEvent(event)`